### DISCUSSION TITLE
Problem auth'ing with BitBucket cloud in 7.7.0

### DISCUSSION THREAD

**Discussion by JBailes** [concourse/concourse#8166] [Help & Support]
Hello. I am trying to get a Concourse installation working in AWS EKS. I currently have all appropriate information in the correct secrets, and verified that I can connect to the container fine.

If I am not logged into BitBucket, I am given the expected prompt to log into BitBucket Cloud when I click on Login. However, if I am already logged into BitBucket or log into BitBucket successfully, I encounter an error of 
"Confirm access to your account

Invalid client_id

This integration is misconfigured. Contact the vendor for assistance."

I've spent several days trying to sort out the problem with this, including trying to utilize BitBucket Cloud as a standard OAuth, of which I receive similar problems with connecting. Would anyone be able to offer advice on what I could do to fix this?

Thank you!

**Comment by taylorsilva** [comment]
I think bitbucket deprecated their old auth api. We use dex and they tracked this issue here: https://github.com/dexidp/dex/issues/2391

looks like it's been fixed (https://github.com/dexidp/dex/pull/2390). We may need to rebase our fork of dex onto theirs to get the fix. 

Some digging needs to happen here to see if we need to update our dex fork. I forget if this happens automatically. What version of dex is our fork based on now? What version of dex did 7.7.0 ship with?

**Comment by JBailes** [comment]
Based on repo commits to cmd, it states 0.4.0 from over a year ago.
